infrastructure/Makefile

```diff
@@ -36,5 +39,32 @@ setup-master-account:
 	@$(eval netAddress=$(shell echo $(list) | awk '{print $$2}'))
 	goal clerk send -a 2000000000000 -f $(netAddress) -t WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE -d $(ALGORAND_DATA)
 
+sandbox-up:
+# clone sandbox git repo: https://github.com/algorand/sandbox.git
+	rm -rf $(SANDBOX_PATH)/
+	git clone https://github.com/algorand/sandbox.git $(SANDBOX_PATH)
+# override .yml with custom config (custom token and port for private network)
+	cat ./sandbox-docker-compose.yml > $(SANDBOX_PATH)/docker-compose.yml
+	$(SANDBOX_PATH)/sandbox up -v
```

Comment:
why do we removing the repository and re-cloning it?

Reply:
I wanted to fetch the latest repo, every time user runs `sandbox-up`.
Upon looking furthur, updating the logic to: a) check if repo exists, if not then clone it. b) after cd, do `git pull` in repo.
Script:
```
export SANDBOX_PATH = ~/sandbox
export REMOTE_REPO = https://github.com/algorand/sandbox.git

[ -d $(SANDBOX_PATH) ] || git clone $(REMOTE_REPO) $(SANDBOX_PATH)
(cd $(SANDBOX_PATH); git pull $(REMOTE_REPO))
```